src/rpc/blockchain.cpp

```diff
@@ -1283,6 +1283,10 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "  \"chainwork\": \"xxxx\"     (string) total amount of work in active chain, in hexadecimal\n"
             "  \"pruned\": xx,             (boolean) if the blocks are subject to pruning\n"
             "  \"pruneheight\": xxxxxx,    (numeric) lowest-height complete block stored\n"
+            "  \"initialblockdownload\":   (boolean) Is in initial block download\n"
```

Comment:
This should probably added next to `verificationprogress`.

Reply:
Hmm.. this will actually conflict with https://github.com/bitcoin/bitcoin/pull/11258/commits/bd9c18171dff394925acf7bbddf773780d1c66bc when we continue to backport bitcoin 0.16, so I'd rather not add this atm.

Reply:
Fair enough, should just be noted to add at some point.